dashboard/management/commands/createkeys.py

```diff
@@ -1,63 +1,36 @@
-import json
 import os
 from datetime import datetime
-from enum import Enum
 
 from Crypto.PublicKey import RSA
 from Crypto.PublicKey.RSA import RsaKey
 from django.core.management.base import BaseCommand, CommandParser
-from jwcrypto.jwk import JWK
```

Comment:
Can we remove this dependency from `requirements.txt` now?

Reply:
I might be able to remove the dependency for `pycryptodome`, too.